### PR TITLE
feat: user account page

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/dapp/src/components/Dashboard/Account/AccountPayments.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+              <tbody>
+                <tr>
+                  <td></td>
+                  <td></td>
+                  <td></td>
+                  <td></td>
+                </tr>
+              </tbody>
+            </AccountPaymentTable>
+            <InvertedButton size={buttonSizes.SMALL}>Add card</InvertedButton>
```
Comment (sruddy) [inline packages/dapp/src/components/Dashboard/Account/AccountPayments.tsx]: This section is coming in the save credit card PR. We're not linking to this page yet.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.8 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**